lifecycleScripts/submodules/getStatus.js

```diff
@@ -5,6 +5,13 @@ var exec = require(path.join(rootDir, "./utils/execPromise"));
 module.exports = function getStatus() {
   return exec("git submodule status", { cwd: rootDir})
     .then(function(stdout) {
+      if (!stdout) {
+        // In the case where we pull from npm they pre-init the submoduels for
```

Comment:
A legendary fight to the death where libgit2 211e11 is overcome by libgit2 c14853